### ISSUE TITLE
Did I get the best psnr

### ISSUE THREAD

**Issue by bing1zhi2** [ychfan/tf_estimator_barebone#9]
appreciate your excellent work..... I'm training WDSR  model on DIV2K_x3 training set ,now the global step is about 2650000: PSNR = 31.0282, global_step = 2650000, loss = 0.01833889. it is increasing at a low speed,and it seems that the psnr can't  reach 32. I  run test code with this model,and feel it not as good as yours ....so did I get the best psnr? what should I do next?  Thanks a lot....

**Comment by ychfan** [comment]
I think 31.02dB is reasonable, if you refer to [EDSR](https://github.com/thstkdgus35/EDSR-PyTorch) which is 30.92dB with more parameters.
If you want to improve the performance, I suggest to make the networks deeper and wider.

**Comment by bing1zhi2** [comment]
thanks
